<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1052</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/Desktop/PyQt5/test.ui/1.jpg);</string>
    </property>
    <column>
     <property name="text">
      <string>学生信息管理</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="background">
      <color>
       <red>255</red>
       <green>255</green>
       <blue>255</blue>
      </color>
     </property>
     <property name="foreground">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>85</green>
        <blue>127</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
       <normaloff>:/newPrefix/timg.jpg</normaloff>
       <normalon>flash.png</normalon>:/newPrefix/timg.jpg</iconset>
     </property>
    </column>
    <item>
     <property name="text">
      <string>切换为成绩信息</string>
     </property>
     <property name="background">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>127</blue>
       </color>
      </brush>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>127</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/back1.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/back1.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>刷新</string>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>127</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/Refresh.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/Refresh.png</iconset>
     </property>
    </item>
    <item>
     <property name="text">
      <string>查询</string>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>127</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/search.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/search.png</iconset>
     </property>
     <item>
      <property name="text">
       <string>按学号</string>
      </property>
      <property name="icon">
       <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
        <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/字母_a.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/字母_a.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>按姓名</string>
      </property>
      <property name="icon">
       <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
        <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/字母_b.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/字母_b.png</iconset>
      </property>
     </item>
    </item>
    <item>
     <property name="text">
      <string>添加</string>
     </property>
     <property name="foreground">
      <brush brushstyle="NoBrush">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>127</blue>
       </color>
      </brush>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/add.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/add.png</iconset>
     </property>
     <item>
      <property name="text">
       <string>手动添加</string>
      </property>
      <property name="icon">
       <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
        <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/添加1.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/添加1.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>文件导入</string>
      </property>
      <property name="icon">
       <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
        <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/添加.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/添加.png</iconset>
      </property>
     </item>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>771</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <column>
     <property name="text">
      <string>学号</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
       <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/Credit_Card.net.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/Credit_Card.net.png</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
       <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/ID_128px_1179915_easyicon.net.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/ID_128px_1179915_easyicon.net.png</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>性别</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
       <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/Sex_.net.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/Sex_.net.png</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>年龄</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
       <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/Calendar.net.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/Calendar.net.png</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>出生年月</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
       <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/Calendar_net.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/Calendar_net.png</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>E-mail</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
       <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/Email_996px_1195279_easyicon.net.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/Email_996px_1195279_easyicon.net.png</iconset>
     </property>
    </column>
    <column>
     <property name="text">
      <string>电话</string>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="icon">
      <iconset resource="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url">
       <normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/phone_.net.png</normaloff>:/newPrefix/Desktop/PyQt5/test.ui/piliang_ico/phone_.net.png</iconset>
     </property>
    </column>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../Pictures/4K/美女30P/路风影视模型素材（5TB容量）.url"/>
 </resources>
 <connections/>
</ui>
